merino-admin/suggestion_providers/models.py

```diff
@@ -0,0 +1,99 @@
+import uuid
+
+from django.db import models
+from polymorphic.models import PolymorphicModel
+
+
+class ProviderConfig(PolymorphicModel):
+    id: models.UUIDField = models.UUIDField(
+        primary_key=True, default=uuid.uuid4, editable=False
+    )
+    exported_name: models.CharField = models.CharField(max_length=256, blank=True, null=True)
+
+    def provider_type(self):
+        return self.get_real_instance().get_type()
+
+    def get_type(self):
+        return '<unknown>'
+
+
+class RemoteSettingsConfig(ProviderConfig):
+    bucket: models.CharField = models.CharField(max_length=256, blank=True, null=True)
+    collection: models.CharField = models.CharField(max_length=256, blank=True, null=True)
+    resync_interval: models.DurationField = models.DurationField(blank=True, null=True)
```

Comment:
These duration fields are currently expected as seconds (field name `resync_interval_sec`) how will the django durations serialize into them?

Reply:
According to the [doc](https://docs.djangoproject.com/en/4.0/ref/models/fields/#durationfield), it's tied to the database of the choice. For SQLite, it's a bigint of microseconds.

In terms of field naming, @mythmon, did you want to stick to the names used in the configuration file?

Reply:
The database representation isn't important here, it's a storage implementation detail. The Python code will represent these as `timedelta` objects, which are similar to Rust's `Duration` type. When this gets serialized to a format for Merino we can choose how it gets formatted.